src/administration/Administration.Service/BusinessLogic/CompanyDataBusinessLogic.cs

```diff
@@ -606,4 +606,23 @@ await foreach (var data in companyCertificateRepository.GetCompanyCertificateDat
             size,
             _settings.MaxPageSize,
             _portalRepositories.GetInstance<ICompanyCertificateRepository>().GetActiveCompanyCertificatePaginationSource(sorting, certificateStatus, certificateType, _identityData.CompanyId));
+
+    /// <inheritdoc />
+    public async Task<(string FileName, byte[] Content, string MediaType)> GetCompanyCertificateDocumentAsync(Guid documentId)
+    {
+        var documentDetails = await _portalRepositories.GetInstance<ICompanyCertificateRepository>()
+            .GetCompanyCertificateDocumentDataAsync(documentId, DocumentTypeId.COMPANY_CERTIFICATE)
+            .ConfigureAwait(false);
+
+        if (!documentDetails.IsExists)
```

Comment:
please rename to Exists
```suggestion
        if (!documentDetails.Exists)
```

Reply:
IsExists changed to Exists